.github/workflows/lint.yml

```
@@ -10,17 +10,17 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
       with:
         submodules: 'recursive'
     - name: Set up Python ${{ matrix.python-version }}
```

actionlint (1.7.7)

property "python-version" is not defined in object type {}

(expression)